modules/serverless-go-template.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-go-template_{context}"]
 = Golang function template structure
 
-When you create a Golang function using the `kn` CLI, the project directory looks like a typical Go project, with the exception of an additional `func.yaml` configuration file.
+When you create a Golang function using the `kn` CLI, the project directory looks like a typical Go project. The only exception is the additional `func.yaml` configuration file, which is used for specifying the image.
```

Comment:
Why use both "Golang" and "Go" to describe the language? I think we should just stick to using the term "Go".